README.md

```diff
@@ -4,14 +4,13 @@ A super minimal implementation of the Solaris backend.
 
 ## Usage
 
-### Build and run
+### Running with hot reloading enabled
 
-```
-docker build -t mocksolaris .
-docker run -p 127.0.0.1:2091:2091 mocksolaris
-```
+You need two terminal sessions:
 
-You may now access the backoffice at [http://localhost:2091/\_\_BACKOFFICE\_\_/](http://localhost:2091/__BACKOFFICE__/).
+1. Run `npm run hot:build`
+2. Run `docker run -p 127.0.0.1:2091:2091 -v $(pwd)/dist:/opt/mockSolaris/dist/ mocksolaris npm run hot:start`.
```

Comment:
Question about this:
Is this something we could have on the npm scripts to avoid having to copy paste such a long command?

`$(pwd)/dist:/opt/mockSolaris/dist/ mocksolaris npm run hot:start` could that be inside the docker file itself or it doesn't make sense at all? 🤔

Reply:
Here: f2fb3d7